src/log.js

```diff
@@ -199,12 +208,13 @@ export class Log {
   }
 
   /**
+   * @param {!Window=} opt_win provided for testing
    * @return {!LogLevel}
    * @private
    */
-  defaultLevelWithFunc_() {
+  defaultLevelWithFunc_(opt_win) {
```

Comment:
@jridgewell Updated this so the `Window` param to `defaultLevelWithFunc_` is optional, and is not passed in by `defaultLevel_` but the test can still pass it in. I plan to simplify it all further eventually, but not in this PR.